metrics.md

```diff
@@ -15,6 +17,24 @@ If there are any additional metrics that you would like to see, please open an i
     * [HTTP Metrics](#http-metrics)
     * [WebSocket Metrics](#websocket-metrics)
 
+# Dashboard
+
+The slinky repo contains a side-car grafana dashboard that can be utilized across deployments. The dashboard is designed to provide a high-level overview of the side-car's health and performance. The dashboard includes various panels that display metrics related to the side-car's health, price feeds, HTTP requests, and WebSocket connections.
```

Comment:
Sometimes Grafana is capitalized, others not.  I think it should be capitalized throughout